input/resources/au-core-server.xml

```diff
@@ -100,9 +100,11 @@
           </div>
           <blockquote>
             <div>
+
               <ol>
-                <li>TBD.</li>
-              </ol>
+                <li>A server <strong>SHALL</strong> reject any unauthorized requests by returning an HTTP 401 "Unauthorized", HTTP 403 "Forbidden", or HTTP 404 "Not Found"</li>
```

Comment:
Remove. This is not clean up, it is proposed content and requires an agreed change request. 

To keep it can be changed to MAY.